src/watchmaker/workers/salt.py

```diff
@@ -254,6 +255,12 @@ def _build_salt_formula(self, extract_dir):
                 filepath=salt_content_file,
                 to_directory=extract_dir
             )
+        else:
+            local_content = os.sep.join((
+                static.__path__[0], 'salt', 'content'))
+            if os.path.exists(local_content):
+                self.log.info('Using local salt content: %s', local_content)
```

Comment:
This wording is slightly confusing, since we're "using" it only in the sense that it is being copied to the location salt expects (or will expect) in the salt config. _Salt_ is not "using" this location. When users run watchmaker, they're generally unaware of this kind of distinction, and I think they would assume this log entry would mean the latter.

Reply:
👍